packages/polling/src/poll.ts

```diff
@@ -241,17 +224,18 @@ export class Poll<T = any, U = any, V extends string = 'standby'>
     this._tick = scheduled;
 
     // Clear the schedule if possible.
-    if (last.interval === Poll.IMMEDIATE) {
-      cancel(this._timeout);
-    } else {
-      clearTimeout(this._timeout);
-    }
+    clearTimeout(this._timeout);
```

Comment:
Should we test for non-undefined timeout id?

```suggestion
    if (typeof this._timeout === 'number') {
      clearTimeout(this._timeout);
    }
```